src/elements/content-explorer/ItemGridThumbnail.scss

```diff
@@ -17,4 +21,9 @@
     position: absolute;
     right: 0;
     top: 0;
+
+    & svg {
```

Comment:
Is the `&` needed here? It should mainly be used to create compound selectors, but this looks like a descendant selector.